app/client/src/PluginActionEditor/components/PluginActionNameEditor.tsx

```diff
@@ -4,7 +4,7 @@ import { usePluginActionContext } from "../PluginActionContext";
 import { useFeatureFlag } from "utils/hooks/useFeatureFlag";
 import { getHasManageActionPermission } from "ee/utils/BusinessFeatures/permissionPageHelpers";
 import { FEATURE_FLAG } from "ee/entities/FeatureFlag";
-import type { ReduxAction } from "ee/constants/ReduxActionConstants";
+import type { ReduxAction } from "../../actions/ReduxActionTypes";
```

Comment:
Is the an issue with using the actions alias path?
```suggestion
import type { ReduxAction } from "actions/ReduxActionTypes";
```

Reply:
Nope. Fixed